lib/auth/auth.go

```diff
@@ -5269,6 +5269,8 @@ func (a *Server) Ping(ctx context.Context) (proto.PingResponse, error) {
 	}
 	features := modules.GetModules().Features().ToProto()
 
+	// DELETE IN 16.0 (just the entire if block, no other changes necessary)
+	// Enabling access monitoring is now done earlier near startup.
```

Comment:
cc @smallinsky deprecating (eventually deleting) `features.IdentityGovernance` felt like the right thing to do. I didn't see how i could keep backwards compat and have `features.IdentityGovernance` only mean IGS. (new auth -> older proxy)

Reply:
What Would you say about somthing like: 
```go
func setAccessMonitoringFeature(ctx context.Context, features  *proto.Features) {
	clientVersionString, ok := metadata.ClientVersionFromContext(ctx)
    if clientSupprotAccessMonitoriungFeature(clientVersionString) {
		features.AccessMonitoring = &proto.AccessMonitoringFeature{
			Enabled: true,
		}
	  } else {
         // TODO Delete in 16 ... 
         features.IdentityGovernance = a.accessMonitoringEnabled
	  }
     }
}   
```

where `clientSupprotAccessMonitoriungFeature` can verify if proxy support proper logic. 

In this solution the `IdentityGovernanceSecurity` is not needed and we can reuse `IdentityGovernance` flag.

Reply:
this was a great suggestion thanks. i wrote a test, and also tested running `v14 proxy` and `master auth`